src/components/apply/formNormalizers.js

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash';
```

Comment:
Just to update (for posterity), those libraries didn't work, we've talked about a different approach that we are taking now.